sync.rb

```diff
@@ -43,7 +44,7 @@ def main
       external_id: content.id,
       url: content.url,
       title: content.title,
-      body: content.html.slice(0...MAX_BODY_LENGTH),
+      body: content.body.slice(0...MAX_BODY_LENGTH),
```

Comment:
I think it's a good idea to rename `MAX_BODY_LENGTH` to `MAX_CONTENT_LENGTH`, too